RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -868,31 +869,6 @@ extension TransactionReview {
 
 		typealias NonFungibleToken = OnLedgerEntity.NonFungibleToken
 
-		func tokenInfo(_ ids: [NonFungibleLocalId], for resourceAddress: ResourceAddress) async throws -> [NonFungibleToken] {
-			if let tokenData = dataOfNewlyMintedNonFungibles[resourceAddress.address] {
-				try extractTokenInfo(tokenData, for: resourceAddress)
```

Comment:
Why is all this going away? Because of the missing data from RET? If so, shouldn't we just comment it out until we get the data again?

Reply:
this will be revisited with the next version, as it will change.